app/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
+    id 'org.jlleitschuh.gradle.ktlint' version "11.3.1"
```

Comment:
This is the actual change: adding the `ktlint` plugin. 
The rest of the file-changes are just after applying the linter behavior.